README.md

```diff
@@ -1,35 +1,53 @@
-# ExercismWinSetup
+# Windows Installer for Exercism Windows CLI
 A project to download and install the latest version of Exercism Client for Windows depending on the architecture.
 
 Exercism is licensed under MIT License. Copyright (c) 2013 Mike Gehard, Katrina Owen
 
 Exercism Repository -> https://github.com/exercism/exercism.io
 
-[![Build status](https://ci.appveyor.com/api/projects/status/6b5cka3nm3r4vh8d?svg=true)](https://ci.appveyor.com/project/blueelvis/windows-installer)
+[![Build status](https://ci.appveyor.com/api/projects/status/6b5cka3nm3r4vh8d?svg=true)](https://ci.appveyor.com/project/rpottsoh/windows-installer)
```

Comment:
Instead of having `blueelvis` or `rpottsoh` or `kytrinyx` in the AppVeyor URL it will be `exercism`.